src/UndoController.js

```diff
@@ -0,0 +1,72 @@
+var diff = require('virtual-dom/diff');
+var patch = require('virtual-dom/patch');
+var virtualize = require('vdom-virtualize');
+var uuid = require('uuid');
+
+module.exports = class UndoController {
+
+  constructor(scribe, attrs){
+    //remove the default undo command as we will be replacing it
+    scribe.commands.undo.execute = function(){};
```

Comment:
I think monkey patching this like this is fine but I wonder if it'd make sense to instead not load the standard undo manager in the first place. Not quite up to date on whether this is currently possible in Scribe. Thoughts? @hmgibson23 @cutandpastey